App/Program.cs

```diff
@@ -1,8 +1,9 @@
-﻿using System;
+﻿using MsgPackV1;
+using System;
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
-#if NETCOREAPP
+#if NET7_0
```

Comment:
We could leave this the way it was. Or at least use the `_OR_LATER` variant of the symbol so that it keeps working when we retarget to NET8_0.